src/extensions/agent_modules/cloud_communication/eg_mqtt_broker_client/src/upd_request_operation.c

```diff
@@ -461,7 +464,7 @@ static void HandleReportResults(ADUC_Retriable_Operation_Context* context, ADUC_
             context->attemptCount = 0;
             context->lastFailureTime = nowTime;
             context->nextExecutionTime = nowTime + 60; // TODO: add random jitter and use retry configuration.
-            AduUpdUtils_TransitionState(ADU_UPD_STATE_IDLEWAIT, updData);
+            AduUpdUtils_TransitionState(ADU_UPD_STATE_IDLEWAIT, updData, context);
```

Comment:
nit: add the + 60 to a configuration that we use for the nextExecution time. 

Also commented out line on line 459. Do we need it still?